apollo-openapi/src/main/java/com/ctrip/framework/apollo/openapi/api/AppOpenApiService.java

```diff
@@ -18,13 +18,22 @@
 
 import com.ctrip.framework.apollo.openapi.dto.OpenAppDTO;
 import com.ctrip.framework.apollo.openapi.dto.OpenEnvClusterDTO;
+import java.util.Collections;
```

Comment:
Appears to be unused?

Reply:
Will remove unused import after this poc code's direction is correct.